en/lessons/advanced/concurrency.md

```diff
@@ -83,7 +83,7 @@ iex> spawn_link(Example, :explode, [])
 ** (EXIT from #PID<0.57.0>) evaluator process exited with reason: :kaboom
 ```
 
-Sometimes we don't want our linked process to crash the current one.  For that we need to trap the exits.  When trapping exits they will be received as a tuple message: `{:EXIT, from_pid, reason}`.
+Sometimes we don't want our linked process to crash the current one. For that we need to trap the exits. To trap the exits, we use the `Process` module's `flag/2` function. This module uses [erlang's process_flag function](http://erlang.org/doc/man/erlang.html#process_flag-2) for the `trap_exit` flag. When trapping exits (`trap_exit` is set to `true`), exit signals will be received as a tuple message: `{:EXIT, from_pid, reason}`.
```

Comment:
Could we change this to: "we use `Process.flag/2`" and "erlang's [process_flag](...) function"?